### PR TITLE
Traffic distribution tests and docs

### PR DESCRIPTION
**Pull request by mendelB**
### What does this PR do?
- Adds Jest tests for the [`TrafficDistribution`](https://github.com/DoSomething/phoenix-next/blob/trafficDistribution-tests-docs/resources/assets/components/TrafficDistribution/TrafficDistribution.js) component. (hope I wasn't being too crazy with those helper methods and setup)
- Adds some minor documentation for the TrafficDistribution. (Hope it's not overkill as it's a small component but felt like a nice thing to have documented?)

### What are the relevant tickets/cards?
#646

### CODE REVIEW

File: docs/SUMMARY.md [hunk 1]
```diff
@@ -11,6 +11,7 @@
 - [Sixpack AB Testing](development/sixpack-ab-testing.md)
 - [Routing](development/routing.md)
 - [Issues &amp; Pull Requests](development/issues-and-pull-requests.md)
+- [Feature Traffic Distribution](development/feature-traffic-distribution)
```
Comment (weerd) [inline docs/SUMMARY.md]: Oh nice!
Maybe do a `Features` subdirectory in Development and have this be a list of docs for specific features?

File: resources/assets/components/TrafficDistribution/TrafficDistribution.test.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  global.Math = mockMath;
+};
+
+const setWithinPool = () => (setMockMath(withinPool));
+const setOutsidePool = () => (setMockMath(outsidePool));
+
+const shallowComponent = () => (
+  shallow(
+    <TrafficDistribution percentage={percentage} feature={feature}>
+      <p>Hey there ole chap!</p>
```
Comment (weerd) [inline resources/assets/components/TrafficDistribution/TrafficDistribution.test.js]: ⬆️ ⬇️ 🎩

File: docs/SUMMARY.md (hunk 1 above)
Comment (mendelB) [reply docs/SUMMARY.md]: Yeah, that's a good idea!

File: docs/development/features/README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Features
+***
+
+We've got loads of cool features we use for development within Phoenix (next). Here's a list of some nifty things you mind find useful for you Phoenix Development Arsenal™.
```
Comment (weerd) [inline docs/development/features/README.md]: ⚔️

**Pull request merged**